features/running/developer_sees_initial_testrun_error_if_suite_not_passing.feature

```diff
@@ -68,13 +68,5 @@ Feature: Use Humbug
         When I run humbug
         Then I should see output containing:
             """
-            Humbug running test suite to generate logs and code coverage data...
-
-            Tests must be in a fully passing state before Humbug is run.
-            Incomplete, skipped or risky tests are allowed.
-            The testing framework reported an exit code of 143.
-            The testing framework ran into a failure or error. Refer to output below.
-            Stdout:
-               > TAP version 13
-               > not ok 1 - Error: FooTest::testAddsNumbers
+            ##teamcity[testFailed
```

Comment:
The teamcity block will have it - but the block varies between test runs, so I just checked that it was output for the scenario..